src/plugins/shared_ux/public/components/empty_state/no_data_views/documentation_link.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiLink, EuiTitle } from '@elastic/eui';
+
+import { FormattedMessage } from '@kbn/i18n-react';
+
+interface Props {
+  documentationUrl: string;
```

Comment:
nit: since this is a `DocumentationLink`, restating it on the prop is a bit redundant.  In addition, given that we're stating it's a link, I'd consider `href`.

```suggestion
  href: string;
```